app/frontend/js/features/RawRecordSlice.js

```diff
@@ -0,0 +1,52 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import { request } from "../utils/request";
+
+export const askNewRawRecord = createAsyncThunk(
+  "fields/askNewRawRecordStatus",
+  async (payload) => {
+    const { transformationDefinitionId, page, record } = payload;
+
+    const response = request
+      .get(
+        `/transformation_definitions/${transformationDefinitionId}/raw_records`,
+        { params: { page: page, record: record } }
+      )
+      .then(function (response) {
```

Comment:
do we need to a catch in case something goes wild?

Reply:
I added a `.rejected` case for when the response fails :)